app/controllers/chargeback_controller.rb

```diff
@@ -171,6 +174,9 @@ def cb_rate_edit
           end
           @sb[:rate_details].each do |detail|
             detail.errors.each { |field, msg| add_flash("'#{detail.description}' #{field.to_s.capitalize} #{msg}", :error) }
+            detail.chargeback_tiers.each do |tier|
+              tier.errors.each { |field, msg| add_flash("'#{detail.description}' #{field.to_s.capitalize} #{msg}", :error) }
```

Comment:
This is identical to 2 lines up, we should see if we can provide a single way to display these errors.